rfd/0058-package-distribution.md

```diff
@@ -0,0 +1,66 @@
+| YUM channel support via URI                        | No (one URI only)          | Yes                                                                                                          | Yes                                                                     | Yes                                            | Yes                                                                               |
+| Overall channel support rating             | 1/5                        | 5/5                                                                                                          | 4/5                                                                     | 2/5                                            | 3/5                                                                               |
+| Channel support notes                      | No support currently       | Can do anything with some reconfiguration                                                                    | Can do anything we care about with some reconfiguration                 | Very limited, no good solution                 | Missing some core features, would require CDN to rewrite HTTP header for requests |
+| Monitoring and alerting support            | In house, poll based only  | In house, poll based only                                                                                    | Webhooks on important events                                            | In house, poll based only                      | In house, poll based only (only supported in AR for Docker images)                |
+| Supports self hosting                      | N/A                        | N/A                                                                                                          | Yes                                                                     | Yes                                            | No                                                                                |
+| Has official Terraform provider            | N/A                        | N/A                                                                                                          | Yes                                                                     | No                                             | Yes                                                                               |
+| Pricing ($)                                | N/A                        | N/A                                                                                                          | $700/month                                                              | $700/month                                     | $0.1/GB/month stored, $0.09/GB/month egress                                       |
```

Comment:
We pay ~$5 for s3 storage of apt/deb/chars and and and we're currently under the cloudfront free tier.  The buckets are <50gb apiece.

I'd be glad to show you around the legacy account where stuff is hosted now.  We're working on moving it to `teleport-prod` which you'll have more access to.